angular_token_auth.js

```diff
@@ -13,6 +13,8 @@
         AUTH_HEADER_PREFIX: 'Token',
         ALLOWED_HOSTS: [],
         COOKIE_PATH: null
+        // Optional settings examples:
+        // STORAGE_METHOD: 'localstorage',
```

Comment:
> Have no default option but give an example with a real value that is not the default

Have no default option but give an example with ~~a real value~~ _all real values_ that ~~is~~ _are_ not the default